### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by bingenito**
Add initial CODEOWNERS as discussed in #1253.

---
THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE FINOS CORPORATE CONTRIBUTOR LICENSE AGREEMENT.
 
THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# fdc3-maintainers will be default reviewers for everything in the repo unless a later match takes precedence
+* @finos/fdc3-maintainers
+
+# Any changes to docs (specs) should also be reviewed by fdc3-editors
+/docs/ @finos/fdc3-editors @finos/fdc3-maintainers
```
Comment (bingenito) [inline .github/CODEOWNERS:5]: @robmoffat Is fdc3-editors a private team at the org level or is it public?

Comment (bingenito) [reply .github/CODEOWNERS:5]: The other issue could be that is that the group as a whole doesn't have write access (individuals within the group might have it as being maintainers/admins).

**Review by kriswest — APPROVED** [review]
LGTM - we may want to add more nuance later (like opening up toolbox to additional owners??)... More likely opening some of the project (outside of /docs, but definitely including the website) to FINOS staff to help with review?

**Comment by bingenito** [comment]
> LGTM - we may want to add more nuance later (like opening up toolbox to additional owners??)... More likely opening some of the project (outside of /docs, but definitely including the website) to FINOS staff to help with review?

When we revisit longer term, one of the requirements is that owners in the file need to have write access which will mean looking at teams and permissions.

**Review by robmoffat — APPROVED** [review]

**Comment by bingenito** [comment]
Fixes #1253

**Pull request merged**